src/application/common/commands-common.ts

```diff
@@ -20,6 +20,7 @@ export namespace CommonCommands {
 
     export const EDIT_UNDO = 'edit_undo';
     export const EDIT_REDO = 'edit_redo';
+    export const SWITCH_HEADER_SOURCE = 'switch_header_source';
```

Comment:
It should not appear in the application extension, it is cpp specific and should belong to the cpp extension